packages/DropdownMenu/src/DropdownMenu.js

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Popover from "@paprika/popover";
+import ContentContainerStyled from "./ContentContainer.styles";
+import Confirmation from "./Confirmation/Confirmation";
+import Divider from "./Divider/Divider";
+import Trigger from "./Trigger/Trigger";
+import Item from "./Item/Item";
+
+const { alignTypes, oneOf, node, string, func } = PropTypes;
+
+const propTypes = {
+  /** Alignment of the dropdown menu */
+  align: oneOf(alignTypes),
+
+  /** Children should consist of <Dropdown.Item /> */
+  children: node.isRequired,
+
+  /** Optional custom classes */
+  className: string,
+
+  /** Render prop for rendering the trigger element that toggles the dropdown */
+  renderTrigger: func.isRequired,
+};
+
+const defaultProps = {
+  align: "bottom",
+  className: "",
+};
+
+const DropdownMenu = props => {
+  const { align, children, className } = props;
+  const [isOpen, setIsOpen] = React.useState(false);
+  const [isConfirming, setIsConfirming] = React.useState(false);
+  const [renderConfirmation, setRenderConfirmation] = React.useState(null);
+
+  const handleCloseMenu = () => {
+    setIsOpen(false);
+    setIsConfirming(false);
+    setRenderConfirmation(null);
+  };
+
+  const handleOpenMenu = () => {
+    setIsOpen(true);
+  };
+
+  const getTriggerStateAndHelpers = () => {
+    return {
+      isOpen,
+      handleOpenMenu,
+    };
+  };
+
+  const handleShowConfirmation = renderConfirmation => () => {
+    setIsConfirming(prevIsConfirmingState => !prevIsConfirmingState);
+    setRenderConfirmation(prevIsConfirmingState => (prevIsConfirmingState ? null : renderConfirmation));
+  };
+
+  const renderContent = () => {
+    if (isConfirming) {
+      return renderConfirmation(handleCloseMenu);
+    }
+
+    return (
+      <ContentContainerStyled isOpen={isOpen}>
```

Comment:
If just the actual styles are exported from `ContentContainer.styles.js`, then this could be something like: `<div css={contentStyles}`.